.github/workflows/microshed-ci.yml

```diff
@@ -59,7 +59,6 @@ jobs:
           - category: COMPLEX
             projects: >
               everything-app
-              everything-jakarta-app
```

Comment:
We should keep both the `everything-app` and `everything-jakarta-app` for testing purposes.
Perhaps we re-name the `everything-app` to `everything-javaee-app`?

Reply:
And then use the 0.9.2 version to run the everything app javaee tests against? Which will never change again IMO? Asking just to clarify what purpose this will serve as I don't see any.

Reply:
The benefit of keeping the integration test around would be:
1. To have a method of testing config if someone where to open up a bug report
2. To have a sample project that users can refer to if they are working with Java EE
3. As we add more features that are only supported on Jakarta EE these test apps will diverge and we can point users to these projects to show what is/isn't supported when using Java EE

Reply:
Hi Kyle,

I agree with you on the first point. The second and third should be well documented or somewhere archived. I will work something out in the codebase.

Reply:
Hi Kyle,

As per your first point. As we drop Java ee support from version 0.9.2. We also aren't concerned about bug fixes for Java ee from now on. The two things just can't simply be combined.

Please proceed by approving this PR. Focussing on keeping Java EE as part of this will shoot us in the foot.

Reply:
And bugfixes can be tested / created by creating a branch of the 0.9.2 tag.

Reply:
Agreed.